kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -33,6 +37,34 @@ with DAG(
         python_callable=start_mlflow_run,
     )
 
+    def create_pipeline_storage(ti, **kwargs):
+        with client.ApiClient(config.load_incluster_config()) as api_client:
+            v1 = client.CoreV1Api(api_client)
+
+            pvc = k8s.V1PersistentVolumeClaim(metadata=k8s.V1ObjectMeta(name=pvc_name, namespace='{{ config.namespace }}'),
+                    spec=k8s.V1PersistentVolumeClaimSpec(access_modes=['ReadWriteOnce'],
+                    resources=k8s.V1ResourceRequirements(requests={'storage':'{{ config.request_storage | safe }}'})))
+
+            v1.create_namespaced_persistent_volume_claim('{{ config.namespace }}', pvc)
+
+    create_pipeline_storage = PythonOperator(
+        task_id='create_pipeline_storage',
+        python_callable=create_pipeline_storage
+    )
+
+    def delete_pipeline_storage(ti, **kwargs):
+        with client.ApiClient(config.load_incluster_config()) as api_client:
+            v1 = client.CoreV1Api(api_client)
+            v1.delete_namespaced_persistent_volume_claim(name=pvc_name, namespace='{{ config.namespace }}')
+
+    delete_pipeline_storage = PythonOperator(
+        task_id='delete_pipeline_storage',
+        python_callable=delete_pipeline_storage,
+        trigger_rule='all_done'
+    )
+
+    volume = k8s.V1Volume(name='storage',persistent_volume_claim=k8s.V1PersistentVolumeClaimVolumeSource(claim_name=pvc_name))
+    volume_mount = k8s.V1VolumeMount(name='storage', mount_path='/data', sub_path=None, read_only=False)
```

Comment:
Consider mounting into /home/kedro/data -> this is where Kedro expects the data in the standard image

Reply:
Thanks for pointing that out.